etc/Syncer.json

```diff
@@ -1,8 +1,8 @@
 {
   "__inputs": [
     {
-      "name": "DS_BIGDATA-CLUSTER",
-      "label": "bigdata-cluster",
+      "name": "DS_HOTEL-BIZREPORT",
```

Comment:
`HOTEL-BIZREPORT `?

Reply:
？？

Reply:
using `HOTEL-xxx` as the name in the template?

Reply:
it's from template importing, I don't know its rules